bin/eapi9-pipestatus.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2024-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+__pipestatus() {
+	local status=( "${PIPESTATUS[@]}" )
+	local s ret=0 verbose=""
+
+	[[ ${1} == -v ]] && { verbose=1; shift; }
+	[[ $# -ne 0 ]] && die "usage: ${FUNCNAME} [-v]"
+
+	for s in "${status[@]}"; do
+		[[ ${s} -ne 0 ]] && ret=${s}
+	done
+
+	[[ ${verbose} ]] && echo "${status[@]}"
+
+	return "${ret}"
+}
```

Comment:
AFAICS the body of this function is an exact copy from `eapi9-pipestatus.eclass`. It would have been nice to mention this in the commit message and give credit to its author. :smile:

Reply:
Done